src/algebra/group_power/basic.lean

```diff
@@ -68,6 +68,8 @@ by rw [pow_succ, pow_one]
 
 alias pow_two ← sq
 
+theorem pow_three {M : Type*} [monoid M] (a : M) : a^3 = a * a * a := by rw [pow_succ', pow_two]
```

Comment:
```suggestion
theorem pow_three (a : M) : a^3 = a * a * a := by rw [pow_succ', pow_two]
```

Reply:
It's certainly subjective but I think it's a useful convenience.

In fact I'd be tempted to add:
```lean
theorem pow_three' (a : M) : a^3 = a * (a * a) := by rw [pow_succ, pow_two]
```
also.

Of course I could be persuaded to drop it if you think it's just clutter.

Reply:
Can you replace the `rw ←pow_three` in your proof by `rw [←pow_two, ←pow_succ']`? If not, you can keep it.

Reply:
> Can you replace the `rw ←pow_three` in your proof by `rw [←pow_two, ←pow_succ']`? If not, you can keep it.

In fact I cannot, because I need to use it inside a `simp only`.